src/test/resources/application-test.yml

```diff
@@ -19,36 +19,36 @@ hmpps.sqs:
   provider: localstack
   queues:
     visit:
-      queueName: ${random.uuid}
-      dlqName: ${random.uuid}
+      queueName: "visit-${random.uuid}"
+      dlqName: "visit-dlq-${random.uuid}"
```

Comment:
Couldn't tell which queue was which when trying to debug a failing test - so prefixed them all.